src/audio_core/audio_renderer.h

```diff
@@ -148,6 +148,14 @@ struct VoiceOutStatus {
 };
 static_assert(sizeof(VoiceOutStatus) == 0x10, "VoiceOutStatus has wrong size");
 
+struct ChannelInfoIn {
+    u32_le id;
+    std::array<float_le, 24> mix;
```

Comment:
```suggestion
    std::array<float_le, 24> mix_volume;
```

Would be a more informative name here.

Reply:
Alright, I took the struct directly from libNX and copied the names.